solr/metadata.csv

```diff
@@ -12,3 +12,15 @@ solr.search_handler.timeouts,gauge,10,timeout,second,Number of responses per sec
 solr.search_handler.time,gauge,10,,,The sum of all request processing times (in milliseconds) per second.,0,solr,search time
 solr.search_handler.avg_time_per_req,gauge,10,millisecond,request,The average time per request.,-1,solr,search time per req
 solr.search_handler.avg_requests_per_sec,gauge,10,request,second,The average number of requests per second.,1,solr,search reqs per sec
+solr.document_cache.lookups,gauge,10,get,second,The total number of cache lookups per second.,0,solr,doc cache gets
```

Comment:
Would be nicer to sort them by metric names.

Reply:
I can sort these. They were previously out of order so I just appended at the bottom. Will update now.

Reply:
I sorted these